fizzbuzz/TestFizzBuzz.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+require_once __DIR__ . '/FizzBuzz.php';
+
+class TestFizzBuzz extends PHPUnit_Framework_TestCase
+{ 
+    /*
+     * Print the numbers 1..100
+     * For multiples of 3, print "Fizz" instead of the number
+     * For multiples of 5, print "Buzz" instead of the number
+     * For multiples of 3 and 5, print "FizzBuzz" instead of the number
+     * Else it returns the number and a new line
+     */
+
+    /**
+     * @dataProvider listOfNumbers
+     */
+    public function testGivenOneNumberThenItReturnsOne($givenNumber, $expectedOutput)
+    {
+        $fizzBuzz = new FizzBuzz();
+        $this->assertEquals($expectedOutput, $fizzBuzz->check($givenNumber));
+    }
+
+    public static function listOfNumbers()
+    {
+        $listOfNumbers = [];
+
+        for ($counter=1; $counter<=100; $counter++) {
```

Comment:
You can also exploit a mathematical property. Run the code on an input N. Count the number of times you see Fizz. It should be N/3. Buzz N/5. FizzBuzz N/15